### PR TITLE
SOC-1195 As a user on desktop, I wan to see nice looking FB-connect button when the label on the button takes two lines

### PR DESCRIPTION
**Pull request by rafalkalinski**
https://wikia-inc.atlassian.net/browse/SOC-1195

### CODE REVIEW

File: front/styles/auth/module/_forms.scss [hunk 1]
```diff
@@ -85,9 +85,13 @@ input[type="submit"],
...
 			display: block;
-			height: 100%;
+			height: 28px;
+			left: 50%;
+			margin-left: -14px;
+			margin-top: -14px;
 			position: absolute;
-			width: 100%;
+			width: 28px;
+			top: 50%;
```
Comment (bkoval) [inline front/styles/auth/module/_forms.scss]: Are you sure this is a right in terms of alphabetical order? :see_no_evil: :hear_no_evil: :speak_no_evil: :imp: :smiling_imp: :name_badge: :worried: :-1: :cry: :disappointed:

**Comment by bkoval** [comment]
LGTM :grinning:

File: front/styles/auth/module/_forms.scss [hunk 2]
```diff
@@ -85,9 +85,13 @@ input[type="submit"],
...
 			display: block;
-			height: 100%;
+			height: 28px;
+			left: 50%;
+			margin-left: -14px;
+			margin-top: -14px;
 			position: absolute;
-			width: 100%;
+			top: 50%;
+			width: 28px;
```
Comment (kvas-damian) [inline front/styles/auth/module/_forms.scss]: What do you think about extracting size to a variable?

**Comment by kvas-damian** [comment]
LGTM

File: front/svg/images/loading.svg [hunk 3]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="30" height="25" viewBox="0 0 30 25"><path d="M15.1 21.5c-4.7 0-8.6-4-8.6-8.9h2.8L4.8 8 .3 12.6h2.9c0 6.8 5.3 12.3 11.9 12.3 3.8 0 7.2-1.8 9.4-4.7l-2.3-2.4c-1.6 2.2-4.2 3.7-7.1 3.7zm13-8.9h-1c0-6.8-5.4-12.4-12-12.4C11.3.2 7.9 2.1 5.7 5l2.4 2.5c1.5-2.3 4.1-3.8 7-3.8 4.8 0 8.7 4 8.7 8.9h-2.9l4.5 4.6 4.5-4.6h-1.8" fill="#FFF" fill-rule="evenodd"/></svg>
\ No newline at end of file
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="-803 405.2 24 20"><style>.st0{fill:#FFFFFF;}</style><g id="Page-1"><g id="Buttons-behaviour"><g id="Imported-Layers"><path id="Fill-1" class="st0" d="M-791 422.4c-3.9 0-7-3.2-7-7.2h2.3l-3.6-3.7-3.6 3.7h2.3c0 5.5 4.4 10 9.7 10 3.1 0 5.9-1.5 7.6-3.8l-1.9-2c-1.4 1.8-3.4 3-5.8 3z"/><path id="Fill-2" class="st0" d="M-780.5 415.3h-.8v-.1c0-5.5-4.3-10-9.7-10-3.1 0-5.9 1.5-7.7 3.9l1.9 2c1.3-1.9 3.4-3.1 5.7-3.1 3.9 0 7 3.2 7 7.2v.1h-2.3l3.6 3.7 3.6-3.7h-1.3"/></g></g></g></svg>
```
Comment (rogatty) [inline front/svg/images/loading.svg]: @rafalkalinski @bkoval this file seems dirty, can you run it through some optimizer?

**Pull request merged**